### ISSUE TITLE
MCU io5 readability

### ISSUE THREAD

**Issue by rusefillc** [andreika-git/hellen-one#148]
![image](https://user-images.githubusercontent.com/48498823/160919263-dfb0abc0-c398-45e1-bf17-aa85ba9f36dc.png)

![image](https://user-images.githubusercontent.com/48498823/160919295-6a54be4d-d2bb-4689-ae6b-2c549fd1484c.png)

![image](https://user-images.githubusercontent.com/48498823/160919328-54840e4d-d61d-444a-9a6c-6b1e37eb876e.png)

**Comment by rusefillc** [comment]
we have out_io1 and io1 which are different things, io1 which is connected to io5, io5 and out_io5 which are different things

that's confusing

let's rename both IO1 and is IO5 which are same thing to GP1 maybe? any other ideas?

**Comment by andreika-git** [comment]
Yes, we can rename IO[1|5]+ to GP1+. Do we need a new mcu revision for this?

**Comment by rusefillc** [comment]
Nope i think same mcu revision perfectly fine as long as zero copper changes

**Comment by rusefillc** [comment]
@Qwerty-OFF please rename the not-"OUT" IO to GP_IOx

![image](https://github.com/user-attachments/assets/ce6daed2-f0d7-4d99-be88-8d672bc99159)
